apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -3257,11 +3253,11 @@ a.btn.btn-xs.btn-link.pull-right {
     top: -4px;
 }
 
-.multiple-unit {
+.mutliple_unit {
```

Comment:
This change is reverted while according to the code design, we've fixed the camel case classes. @prateek-giddh

Reply:
Done